src/lib/plugin-loader.js

```diff
@@ -56,6 +56,21 @@ export default function loadPlugin<T>(
     // try local plugins first
     plugin = tryLoad(Path.resolve(__dirname + '/../plugins', pluginId));
 
+    // custom plugin directory
```

Comment:
Please update the comment above, just it's a bit outdated according to the new logic.